GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Submodules.cs

```diff
@@ -91,9 +96,33 @@ protected override string NodeName()
 
             public void Open()
             {
+                if (Info?.Detailed?.RawStatus is not null)
+                {
+                    UICommands.BrowseSetWorkingDir(Info.Path, ObjectId.WorkTreeId, Info.Detailed.RawStatus.OldCommit);
+                    return;
+                }
+
                 UICommands.BrowseSetWorkingDir(Info.Path);
             }
 
+            public void OpenGE()
+            {
+                var first = string.IsNullOrWhiteSpace(Info?.Detailed?.RawStatus?.OldCommit?.ToString())
+                    ? string.Empty
+                    : $",{Info?.Detailed?.RawStatus.OldCommit}";
+
+                var process = new Process
+                {
+                    StartInfo =
+                    {
+                        FileName = Application.ExecutablePath,
+                        Arguments = $"browse -commit={ObjectId.WorkTreeId}{first}",
+                        WorkingDirectory = Info.Path.EnsureTrailingPathSeparator()
+                    }
+                };
+                process.Start();
```

Comment:
This needs to be adjusted as per https://github.com/gitextensions/gitextensions/pull/8738

Reply:
of course, I have waited to rebase this partly due to that PR